src/controllers/controllers/controller.py

```diff
@@ -72,9 +98,14 @@ def __init__(self, node_name, car_name, step_length, lidar_points = 10):
 
         self.timer = self.create_timer(step_length, self.timer_cb)
         self.timer_future = Future()
+        
+        # Lidar processing 
+        self.ae_lidar_model = LidarConvAE()
+        # self.ae_lidar_model = BetaVAE1D(1,10,beta=4)
+        self.ae_lidar_model.load_state_dict(torch.load("/home/anyone/autonomous_f1tenth/lidar_ae_ftg_rand.pt"))
```

Comment:
This line breaks my program even with the file lidar_ae_ftg_rand.pt saved in the same location as pr'd. Regardless, breaking errors based on whether ae files are available should not be included (as it shouldn't impact trying to run traditional control with no aes). This could be fixed with a try except clause to only kill the program if you are trying to use ae and it doesn't load